en_US/advanced/relup.md

```diff
@@ -1,312 +1,173 @@
-
-```bash
-
-$ emqx install 4.2.1
-
-Release 4.2.1 is already installed and current, making permanent.
-Made release permanent: "4.2.1"
-
+emqx-[EMQX Version]-[OTP Version]-[OS Type]-[Arch].zip
 ```
 
-## Downgrade to previous versions
-
-If you find a problem and want to roll back after the upgrade, you can execute the version downgrade command.
-For example, the following example will roll back EMQX to version 4.2.0:
-
-```bash
-
-$ emqx downgrade 4.2.0
-
-Release 4.2.0 is marked old, switching to it.
-Installed Release: 4.2.0
-Made release permanent: "4.2.0"
-
-```
-
-## Delete versions
-
-After the system has been running stably for a period of time, if you decide to delete an old version, you can execute the version uninstall command.
-For example, the following example will uninstall the old version 4.2.0:
-
-```bash
-
-$ emqx uninstall 4.2.0
-
-Release 4.2.0 is marked old, uninstalling it.
-Uninstalled Release: 4.2.0
-
-```
+Take `emqx-4.4.16-otp24.3.4.2-1-ubuntu20.04-amd64.zip` for example:
 
+- EMQX Version(`4.4.16`): EMQX version number. Please ensure that all the digits except the last one (here is 16) in the version number must be the same as that of the installed EMQX.
+- OTP Version(`otp24.3.4.2-1`): Erlang OTP version number. Please ensure that the first digit (here is 24) must be the same as that of the installed EMQX.
+- OS Type(`ubuntu20.04`): OS type. Please ensure that it is same as the OS Type of the installed EMQX.
+- Arch(`amd64`): Type of the architecture. Please ensure that it is the same as the arch type of the installed EMQX.
 {% endemqxce %}
 
 {% emqxee %}
+Go to [EMQX Enterprise](https://www.emqx.com/en/try?product=enterprise) for the version and the OS you want, and then select the **"zip"** in the "Install Method".
 
-Since version 4.2.0, EMQX Enterprise supports hot upgrade.
-
-By using the hot upgrade feature, users can quickly and safely upgrade the EMQX Enterprise in the production environment, and avoid the decrease in system availability caused by restarting the service.
+The Zip package name format of EMQX Enterprise version 4.4 is:
 
-::: tip
-Currently EMQX for Windows and MacOSX does not support hot upgrade feature.
-:::
+```
+emqx-[EMQX Type]-[EMQX Version]-[OTP Version]-[OS Type]-[Arch].zip
+```
 
-::: warning
-EMQX only allows hot upgrade between patch versions (the third digit of the version number).
-i.e. upgrading from x.y.z to x.y.(z+N) is allowed, but upgrading from x.y to (x+N).(y+M) is not allowed.
-Please operate with caution.
-:::
+Take `emqx-ee-4.4.16-otp24.3.4.2-1-ubuntu20.04-amd64.zip` for example:
 
-::: warning
-Hot upgrade between EMQX (opensource) and EMQX Enterprise is not allowed, please operate with caution.
-:::
+- EMQX Type(`ee`): EMQX Type. `ee` stands for EMQX Enterprise, Open source version does not have this field. Please make it is the same as the installed EMQX Type.
```

Comment:
should we just omit this part ? "`ee` stands for EMQX Enterprise, Open source version does not have this field."

Reply:
I just want to warn the user do not download the open source packages. I will rephrase it a bit more.